frodo.js

```diff
@@ -1,68 +1,70 @@
-const uniqueSortedArray = [...new Set(array)].sort();
```

Comment:
SCRIPT-I'm impressed with your use of design patterns here, it really enhances the code organization.